README.md

```diff
@@ -1 +1,54 @@
-pretend this readme is as interesting as the website
+![Logo](https://pbs.twimg.com/profile_images/1373353833330475008/rKWjiXET_400x400.png)
+
+
+# Bit Project Website
+
+This GitHub repo is the home 🏠 of the Bit Project website. 
+Here at the Bit Project We equip students with technical 
+superpowers by providing cutting-edge, open source learning 
+experiences to prepare students for careers in tech. 
+
+
+## Serverless Camp (A Bit Project Program) Sponsors
+
+ - ⭐️ [Microsoft](https://www.microsoft.com/en-us/)
+ - ⭐️ [Courier](https://www.courier.com/)
+
+
+
+## Projects From Former Bit Project Students
+
+Too see exactly the kind of work we do at Bit Project, 
+feel free to check out some student projects from 
+
+- Grace built [EcoMaps](https://www.bitproject.org/projects/reducing-carbon-emissions) and reduced carbon emissions 🌱
+- Jakub [analyzed vehicle telemetry data](https://www.bitproject.org/projects/vehicle-telemetry-sigma-link) in real-time 🚗
+- Kaley developed [TimeBlocker](https://www.bitproject.org/projects/saving-time-with-timeblocker) and Boosted Productivity 📚
+- Fifi [saved money shopping on Amazon](https://www.bitproject.org/projects/saving-money-with-serverless) with Serverless 🛒
+
+
+## Meet the Team!
+
+- 💙 Shreya Gupta, Founder (Operations) [Github Link](https://github.com/shreythecray)
+
+- 💙 Daniel Kim, Founder (Technology) [Github Link](https://github.com/lazyplatypus)
+- 💙 Ganning Xu, Director of Engineering [Github Link](https://github.com/ganning127)
+- 💙 Emily Chen, Director of Engineering [Github Link](https://github.com/emsesc)
+- 💙 Misha Patel, Director of Partnerships [Github Link](https://www.github.com/octokatherine)
+- 💜 Beatrix Cendana, Student Community Manager [Github Link](https://github.com/beatrixcendana)
+
+
+## FAQ
+
+#### Are your programs free?
```

Comment:
Can you make these a dropdown? Here's the format for it:
```
<details>
  <summary>Want to ruin the surprise?</summary>
  <br>
  Well, you asked for it!
</details>
```

**Example:**
<details open>
<summary>Want to ruin the surprise?</summary>
<br>
Well, you asked for it!
</details>

Reply:
Here's the source! https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61